antora-ui-camel/src/css/frontpage.css

```diff
@@ -257,6 +257,16 @@ section.frontpage h1 {
     height: 10rem;
   }
 
+  .split {
+    padding: 0 2rem;
+    flex: 100%;
+    text-align: center;
```

Comment:
Not sure about aligning the text. If you'd like to center the 20 years of ASF image perhaps it's better to change the `.split img` to `display: block; margin: auto`.

Currently
![Screenshot_2020-04-01 Home - Apache Camel(1)](https://user-images.githubusercontent.com/1306050/78134971-52326a80-7421-11ea-9168-b69fb773b2bf.png)

Non centered text and centered image
![Screenshot_2020-04-01 Home - Apache Camel(2)](https://user-images.githubusercontent.com/1306050/78134977-53639780-7421-11ea-869d-9e253302bea4.png)

Reply:
Okay, if left-aligned is preferred I'll change that!